in_toto/models/common.py

```diff
@@ -102,6 +102,7 @@ def verify_signatures(self, keys_dict):
       except:
         raise SignatureVerificationError(
             "Signature key not found, key id is '{0}'".format(keyid))
-      if not keys.verify_signature(key, signature, self.payload):
+      if not securesystemslib.keys.verify_signature(
```

Comment:
I am not sure, if the `<Exceptions>` docstring of a function should also list the possible exceptions of the functions it is calling.